tests/basic/podmetric_test.go

```diff
@@ -41,12 +38,58 @@ var _ = Describe("{PodMetricFunctional}", func() {
 		// shared test function for pod metric functional tests
 		sharedTestFunction := func() {
 			It("has to fetch the logs from loggly", func() {
-				log.InfoD("fetching logs from loggly")
-				resp, code, err := getLogglyData("q=tag:meteringData&from=-60m&until=now&size=1")
+				meteringInterval := 1 * time.Minute
+				log.InfoD("Testing with metering interval %v", meteringInterval)
+
+				log.InfoD("Getting cluster ID")
+				clusterUUID, err := getClusterID()
+				log.FailOnError(err, "Failed to get cluster id data")
+
+				log.InfoD("Fetching logs from loggly")
+				meteringData, err := getMeteringData(clusterUUID, meteringInterval)
+				log.FailOnError(err, "Failed to get metering data")
+
+				existsData := len(meteringData) > 0
+				dash.VerifyFatal(existsData, true, "there should be metering data originally in loggly")
+				initialPodHours := getLatestPodHours(meteringData)
+				log.InfoD("Latest pod hours before starting app: %v", initialPodHours)
+
+				log.InfoD("Deploy applications")
+				contexts = make([]*scheduler.Context, 0)
+				for i := 0; i < Inst().GlobalScaleFactor; i++ {
+					contexts = append(contexts, ScheduleApplications(fmt.Sprintf("%s-%d", namespacePrefix, i))...)
+				}
+
+				log.InfoD("Validate applications")
+				ValidateApplications(contexts)
+
+				waitDuration := meteringInterval + 30*time.Second
+				log.InfoD("Wait %v for previous interval to go through", waitDuration)
+				time.Sleep(waitDuration)
+
+				log.InfoD("Wait %v for a new interval to go through", waitDuration)
+				time.Sleep(waitDuration)
+
+				log.InfoD("Check metering data is accurate")
+				meteringData, err = getMeteringData(clusterUUID, meteringInterval)
+				log.FailOnError(err, "Failed to get metering data")
+				existsData = len(meteringData) > 0
+				dash.VerifyFatal(existsData, true, "there should be metering data in loggly")
+				for _, md := range meteringData {
+					dash.VerifyFatal(md.ClusterUUID, clusterUUID, "this cluster should have data now")
+				}
+
+				log.InfoD("Check pod hours is correct")
+				expectedAppPodHours, err := getExpectedPodHours(contexts, meteringInterval)
+				log.FailOnError(err, "Failed to get expectedAppPodHours")
+				expectedPodHours := (float64(expectedAppPodHours) / 60) + initialPodHours
+				log.InfoD("Estimated pod hours for this app is %v", expectedPodHours)
+				log.InfoD("Estimated total pod hours is %v", expectedPodHours)
```

Comment:
Remove one of these prints, its same thing, unless you mean to print something different here